_posts/2024-12-17-release-0_9_0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Kroxylicious release 0.9.0"
+date:   2024-12-17 00:00:00 +0000
+author: "Sam Barker"
+author_url: "https://github.com/SamBarker"
+categories: kroxylicious releases
+tags: releases kroxylicious kafka3.9
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.9.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.9.0). This release introduces support for Apache Kafka 3.9. 
+
+We generally expect Kroxylicious to be both forwards- and backwards-compatible across Kafka versions just like the Apache Kafka client and broker. While we tested with pre-release versions of Apache Kafka 3.9 we discovered after it was released that we had missed a case in the protocol version negotiation. When the `APIVersionsRequest` version was updated we were unable to decode the response. In the 0.9.0 release we adopt the same behaviour as the Kafka broker and respond with API version 0 if we do not support the protocol version requested by the client - we have also expanded our test coverage to get earlier warnings of similar issues in the future. 
```

Comment:
nit: capitalisation
nit: it is the ApiVersions RPC that is the versioned thing, rather than APIVersionsRequest

```suggestion
We generally expect Kroxylicious to be both forwards- and backwards-compatible across Kafka versions just like the Apache Kafka client and broker. While we tested with pre-release versions of Apache Kafka 3.9 we discovered after it was released that we had missed a case in the protocol version negotiation. When the version of the `ApiVersions` RPC was updated we were unable to decode the response. In the 0.9.0 release we adopt the same behaviour as the Kafka broker and respond with API version 0 if we do not support the protocol version requested by the client - we have also expanded our test coverage to get earlier warnings of similar issues in the future. 
```